cluster/cluster.go

```diff
@@ -77,7 +77,10 @@ func (c *Cluster) DeployControlPlane(ctx context.Context) error {
 	if len(c.Services.Etcd.ExternalURLs) > 0 {
 		log.Infof(ctx, "[etcd] External etcd connection string has been specified, skipping etcd plane")
 	} else {
-		if err := services.RunEtcdPlane(ctx, c.EtcdHosts, etcdNodePlanMap, c.LocalConnDialerFactory, c.PrivateRegistriesMap, c.UpdateWorkersOnly, c.SystemImages.Alpine); err != nil {
+		enableBackup := c.Services.Etcd.Backup
```

Comment:
would be nice to create a wrapper for etcdBackup, and pass it as a single variable to RunEtcdPlane